types/node/http.d.ts

```diff
@@ -479,11 +491,26 @@ declare module 'http' {
         setTimeout(msecs: number, callback?: () => void): this;
         /**
          * Sets a single header value for the header object.
+         * If the header already exists in the to-be-sent headers, its value will be replaced. Use an array of strings to send multiple headers with the same name.
          * @since v0.4.0
          * @param name Header name
          * @param value Header value
          */
         setHeader(name: string, value: number | string | ReadonlyArray<string>): this;
+        /**
+         * Append a single header value for the header object.
+         *
+         * If the value is an array, this is equivalent of calling this method multiple times.
+         *
+         * If there were no previous value for the header, this is equivalent of calling `outgoingMessage.setHeader(name, value)`.
+         *
+         * Depending of the value of `options.uniqueHeaders` when the client request or the server were created,
+         * this will end up in the header being sent multiple times or a single time with values joined using `; `.
+         * @since v18.3.0, v16.17.0
+         * @param name Header name
+         * @param value Header value
+         */
+        appendHeader(name: string, value: number | string | ReadonlyArray<string>): this;
```

Comment:
Isn't `value` a `string`? Why `number`?
https://nodejs.org/docs/latest-v18.x/api/http.html#outgoingmessageappendheadername-value

Reply:
I copied the definition from `setHeader`. Maybe both should be updated?

Reply:
As pointed out by @Semigradsky, from the Node docs, `value` is `any` in `setHeader`, but should be `string | string[]` in `appendHeader`. Please update this accordingly.